packages/suite/src/components/suite/layouts/WelcomeLayout/WelcomeLayout.tsx

```diff
@@ -47,6 +47,7 @@ const Expander = styled.div`
     justify-content: center;
     align-items: center;
     flex: 1;
+    margin-top: calc(${spacingsPx.xxxxl} * 2);
```

Comment:
Not sure maybe just place `96px`? If its not semantic. (WDYT @jvaclavik ?)

Reply:
I agree with using just `96px`